tests/async-unit/test_intake_outbox.py

```
@@ -753,7 +757,7 @@ def test_claim_is_renewed_before_the_rpc(self):
         denial = _make_denial(8142)
         row = _pending(denial, intake_outbox.INTAKE_STARTED)
         claims = intake_outbox.claim_batch()
-        (pk, token) = claims[0]
+        pk, token = claims[0]
```

Ruff (0.16.3)

[warning] Unpacked variable `token` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)